server/channels/app/import_functions.go

```diff
@@ -1272,10 +1273,60 @@ func (a *App) importReplies(rctx request.CTX, data []imports.ReplyImportData, po
 	return nil
 }
 
+func compareFilesContent(fileA, fileB io.Reader, bufSize int64) (bool, error) {
+	aHash := sha1.New()
+	bHash := sha1.New()
```

Comment:
Should we use sha256 instead? The [docs](https://pkg.go.dev/crypto/sha1#pkg-overview) clearly recommend not to use it. I know we are just using for non security purposes, but still feels like good security practice to use 256 instead.

Reply:
I am open to it but I suspect SHA1 to still be the best performing algo and given the checksum use case here I feel it's still reasonable to use it. No strong opinions though but would expect a benchmark to compare.

Reply:
I was searching for benchmarks. I can cook up some tomorrow. Let's see.

Reply:
It should also be straightforward to run a couple through `BenchmarkCompareFilesContent`. Probably not a big deal when compared to io times but we can check.